python/cuml/model_selection/__init__.py

```diff
@@ -0,0 +1,4 @@
+from ._split import train_test_split
```

Comment:
Let's switch this to an absolute import [as recommended](https://www.python.org/dev/peps/pep-0008/#imports) by PEP8.